app/views/idv/in_person/ready_to_verify/show.html.erb

```diff
@@ -232,24 +232,21 @@
 <p><%= t('in_person_proofing.body.expect.info') %></p>
 
 <p class="margin-top-3 margin-bottom-4">
-  <% if @presenter.service_provider_homepage_url.present? %>
-    <%= render ClickObserverComponent.new(event_name: 'IdV: user clicked sp link on ready to verify page') do %>
-      <%= t(
-            'in_person_proofing.body.barcode.return_to_partner_html',
-            link_html: link_to(
-              t(
-                'in_person_proofing.body.barcode.return_to_partner_link',
-                sp_name: @presenter.sp_name,
-              ),
-              @presenter.service_provider_homepage_url,
-            ),
-          ) %>
-    <% end %>
-  <% else %>
+  <% if @presenter.service_provider_homepage_url.blank? %>  
     <%= t('in_person_proofing.body.barcode.close_window') %>
   <% end %>
 </p>
 
-<%= render PageFooterComponent.new do %>
+<%= render PageFooterComponent.new(class: 'site') do %>
```

Comment:
Can you explain the purpose of adding this class?

Reply:
@aduth Sure - I needed some styling to add some spacing around the new anchor link and the existing one. Specifically, I wanted to use `display: flex` with a column flex direction. I saw this class has exactly that so I thought it would be a use case for this.

Reply:
I think if you need spacing, I'd suggest adding a top margin or padding on the second link, using [design system utilities classes](https://designsystem.digital.gov/utilities/margin-and-padding/).

There's two problems I can see with using this class:

- `display: flex` will cause the links to span the full width of the container, so they're clickable outside the area of the link itself, which is unexpected.
- The `site` class is meant for the page `<body />`, and using it here creates a risk that if we add styles to the page body in the future, we may unintentionally affect styles for this page as well.

Worst case, if we really wanted flex styling, I'd suggest using [design system utility classes](https://designsystem.digital.gov/utilities/flex/) to add those desired styles.